server/controllers/BookController.js

```diff
@@ -75,6 +75,62 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* Edit Books
+* @name editBook
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+* @memberof BookController
+*/
+  static async editBook(request, response) {
```

Comment:
`updateBook` sounds better

Reply:
EditBook is okay too, is all about using a variable that will make someone understand what you're doing.

Reply:
But you will agree that updateBook sounds alot better

Reply:
EditBook is also okay too..